app/views/govuk_publishing_components/components/_share_links.html.erb

```diff
@@ -10,15 +10,15 @@
   columns ||= false
   square_icons ||= false
 
-  brand ||= false
-  brand_helper = GovukPublishingComponents::AppHelpers::BrandHelper.new(brand)
+  black_icons ||= false
+  black_links ||= false
```

Comment:
Do we need `black_icons` and `black_links` to be separate options? The use case we're working towards would have both of these options enabled together, is that right?

Reply:
@andysellick `black_icons` will be used on all organisations, they will have a black icon and blue links. Whereas the no 10 page will have black icons and black links, which is why both are separate

Reply:
That makes sense, thanks 👍